.github/workflows/awesome_workflow.yml

```diff
@@ -100,5 +102,17 @@ jobs:
       - uses: actions/checkout@v3
         with:
           submodules: true
-      - run: cmake -B ./build -S .
-      - run: cmake --build build
+      - run: |
+          cmake -B ./build -S .
+          cmake --build build
+      - name: Label on PR fail
+        uses: actions/github-script@v6
+        if: ${{ failure() && matrix.os == 'ubuntu-latest' && github.event_name == 'pull_request' }}
```

Comment:
Same comment as the comment in the PR in C++ repo

> Should we not have a scheduled job for checking for failures on all of the CI's instead of just checking for `failure` on this Action?

Reply:
https://github.com/TheAlgorithms/C-Plus-Plus/pull/2455#discussion_r1183037035

Reply:
Actually, we should see if there's a possible way to do this. I'll be merging this for now, though.